DOCS-3056: Edit arm motion planning section (#3611)
viamrobotics/docs: docs/components/arm/_index.md

{{< readfile "/static/include/components/apis/generated/arm-table.md" >}}

## Motion planning with your arm

> Each arm model provided by Viam wraps the arm manufacturer's API with the Viam [arm API](#api).

Reviewer: This isn't necessarily true. Some arms directly interact with servos, or something else.

> Each arm model provided by Viam wraps the arm manufacturer's API with the Viam [arm API](#api).

Author: ```suggestion
```
Actually this should probably go too since this is how many components work and is kind of the point.

> The arm API sends requests for the arm to move to a set of joint positions.

Reviewer: "and also reports the arm's current joint positions"

Author: ```suggestion
The arm API sends requests for the arm to move to a set of joint positions, and reports the arm's current joint positions.
```

{{< readfile "/static/include/components/apis/generated/arm-table.md" >}}

## Motion planning with your arm

Each arm model provided by Viam wraps the arm manufacturer's API with the Viam [arm API](#api).

> The arm API handles turning the arm on and off, sending requests for the arm to move to a set of joint positions, and engaging brakes, if supported.

Author: ```suggestion
The arm API handles turning the arm on and off, and sending requests for the arm to move to a set of joint positions.
```

Viam's motion service provides an [API for moving the end of the arm to a given position, around any obstacles](/services/motion/#api).

> Some arm manufacturers build inverse kinematics into their software, but the Viam motion service bypasses it, so all arms are controlled by the motion service in the same way.

Author: ```suggestion
```

For each arm model, there is a JSON file that describes the [kinematics parameters of the arm](/internals/kinematic-chain-config/#kinematic-parameters).

> When you configure an arm model, the arm driver will load and parse the kinematics file for the [frame system](/services/frame-system/) service to use.

Author: ```suggestion
When you configure an arm model, the arm driver parses the kinematics file for the [frame system](/services/frame-system/) service to use.
```
